app/views/order_items/_order_item.text.erb

```diff
@@ -0,0 +1 @@
+<%= sprintf("%2d x %s", order_item.quantity, order_item.menu_item.description) %>
```

Comment:
I had thought about that and forgot to remove this restriction - will do so now.